@stellar/anchor-tests/src/schemas/sep24.ts

```diff
@@ -1,3 +1,123 @@
+// =========================== > CONFIG FILE schema =========================== //
+const initialConfigSchema = {
+  type: "object",
+  properties: {
+    accountHolder: {
+      type: "object",
+      properties: {
+        accountAddress: { type: "string" },
+        accountSignerSecretKey: { type: "string" },
+      },
+      required: ["accountAddress", "accountSignerSecretKey"],
+    },
+  },
+  required: ["accountHolder"],
+};
+
+const pendingDepositConfigSchema = {
+  depositPendingTransaction: {
+    type: "object",
+    properties: {
+      status: {
+        type: "string",
+        enum: [
+          "pending_anchor",
+          "pending_external",
+          "pending_stellar",
+          "pending_trust",
+          "pending_user",
+          "pending_user_transfer_start",
+          "pending_user_transfer_complete",
+        ],
+      },
+      id: { type: "string" },
+    },
+    required: ["status", "id"],
+  },
+};
+
+const completedDepositConfigSchema = {
```

Comment:
Can we make all this schemas more strict?
[Deposit](https://github.com/stellar/kotlin-wallet-sdk/blob/main/wallet-sdk/src/main/kotlin/org/stellar/walletsdk/anchor/AnchorTransaction.kt#L40)
[Withdrawal](https://github.com/stellar/kotlin-wallet-sdk/blob/main/wallet-sdk/src/main/kotlin/org/stellar/walletsdk/anchor/AnchorTransaction.kt#L66)
Both pending and completed. 
Everything that is not nullable should be mandatory.
This schema matches SEP-24 protocol and polaris implementation

Reply:
Sure, let me review it

Reply:
@Ifropc I think you were looking at the Config File schema, if you look at the Transaction schema you'll see it's already as strict as the kotlin one. I realized the schema file was looking a bit confusing so I just reorganized/rewrote some stuff to make it more clear. I've just put the Transaction schema at the top of the file and the Config File schema at the bottom of the file to avoid confusion. Please let me know if it's looking good now - thanks!

Reply:
I've left some extra comments to highlight them